lua/entities/cap_doors_contr.lua

```diff
@@ -102,7 +102,7 @@ function ENT:PressButton()
 				self.Entity:EmitSound(self.Sounds.PressDest,100,math.random(90,110));
 			elseif frame.Lockdown then
 				no_sound = true
-				self.Entity:EmitSound(self.Sounds.LockedDest,100,math.random(90,110));
+				self.Entity:EmitSound(self.Sounds.LockedDest,100,100); -- dont put random pitch, it sounds weird then
```

Comment:
In SGU, when they locked the doors for Chloe, she pushed a button to try open the door. It was locked, and the sound always sounded the same